app/src/main/java/com/codesoom/assignment/dto/ErrorResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.codesoom.assignment.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class ErrorResponse {
+
+    private String message;
```

Comment:
이건 `final`로 선언해도 괜찮아 보여요.
```suggestion
    final private String message;
```